content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md

```diff
@@ -0,0 +1,274 @@
+Para descoberta baseada em token, permite associar-se ao cluster sem fixação da
+autoridade de certificação (opção --discovery-token-ca-cert-hash).
+</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--dry-run</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+Não aplica as modificações; apenas imprime as alterações que seriam efetuadas.
+</p></td>
+</tr>
+
+<tr>
+<td colspan="2">-h, --help</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>ajuda para join</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--ignore-preflight-errors strings</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Uma lista de verificações para as quais erros serão exibidos como avisos. Exemplos: 'IsPrivilegedUser,Swap'. O valor 'all' ignora erros de todas as verificações.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--node-name string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Especifica o nome do nó.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--patches string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Caminho para um diretório contendo arquivos nomeados no padrão "target[suffix][+patchtype].extension". Por exemplo, "kube-apiserver0+merge.yaml" ou somente "etcd.json". "target" pode ser um dos seguintes valores: "kube-apiserver", "kube-controller-manager", "kube-scheduler", "etcd". "patchtype" pode ser "strategic", "merge" ou "json" e corresponde aos formatos de patch suportados pelo kubectl. O valor padrão para "patchtype" é "strategic". "extension" deve ser "json" ou "yaml". "suffix" é uma string opcional utilizada para determinar quais patches são aplicados primeiro em ordem alfanumérica.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--skip-phases strings</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Lista de fases a serem ignoradas.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--tls-bootstrap-token string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+Especifica o token a ser utilizado para autenticar temporariamente com a camada de gerenciamento do Kubernetes durante
+o processo de associação do nó ao cluster.
+</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--token string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+Utiliza este token em ambas as opções discovery-token e tls-bootstrap-token quando tais valores não são informados.
+</p></td>
+</tr>
+
+</tbody>
+</table>
+
+
+
+### Opções herdadas dos comandos superiores
+
+   <table style="width: 100%; table-layout: fixed;">
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--rootfs string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>[EXPERIMENTAL] O caminho para o sistema de arquivos raiz 'real' do host.
```

Comment:
I think the 'real' it's the host. "sistema de arquivos raiz do host 'real'." instead of "sistema de arquivos raiz 'real' do host."

Reply:
I think "real" refers to the filesystem and not the host in this context. I also tried my best to keep this consistent with the other kubeadm docs.